src/modules/CourseDetail/index.tsx

```diff
@@ -109,10 +110,11 @@ export function CourseDetailPage(props: { data: GetCourseResponse }) {
           <DescriptionTitle variant="subtitle1">เงื่อนไขรายวิชา</DescriptionTitle>
           <Typography variant="h6">{course.courseCondition}</Typography>
         </Grid>
-        {course.courseDesc && (
+        {(course.courseDescTh || course.courseDescEn) && (
           <Grid item xs={12} sm={12}>
             <DescriptionTitle variant="subtitle1">คำอธิบายรายวิชา</DescriptionTitle>
-            <Typography variant="h6">{course.courseDesc}</Typography>
+            {course.courseDescTh && <CourseDescription variant="h6">{course.courseDescTh}</CourseDescription>}
+            {course.courseDescEn && <CourseDescription variant="h6">{course.courseDescEn}</CourseDescription>}
```

Comment:
ถ้าจะทำ 2 ภาษา มี 2 ทางเลือก 
1. ทำให้เป็น 2 ภาษาทั้งระบบแล้วเลือก description มาแค่อันเดียว
2. เขียนคำอธิบายรายวิชา 2 หัวข้อ
```
คำอธิบายรายวิชา (ภาษาไทย)
ความหมาย ประเภท และการเกิดของสาธารณภัย การเฝ้าระวังเพื่อช่วยเหลือตนเองและผู้อื่นให้พ้นจากสาธารณภัย ผลกระทบของสาธารณภัย กระบวนการบริหารจัดการดูแลช่วยเหลือและฟื้นฟูสภาพแวดล้อมทางธรรมชาติและสังคม รวมทั้งผู้ประสบภัย

คำอธิบายรายวิชา (อาษาอังกฤษ)
Difinition, types, and occurrences of public disasters; monitoring public disasters for the benefit of the individual and public in order to avoid future disasters; impact of the incidents; management process to alleviate the hardship and to recreate natural and social environments as well to rehabilitate affected invididuals.
```

เอา desc en มาไว้พร้อมกับ desc th โดยที่ context เป็นภาษาไทยมันแปลก ๆ

Reply:
2 ภาษาทั้งเว็บน่าจะอีกนานมาก ๆ อะ

Reply:
done